helpers/3p/utils/lib/deepMatches.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Recursively compare objects
+ */
+const typeOf = require('./kindOf');
+
+function deepMatches(val, value) {
+    if (typeOf(val) === 'object') {
+        if (Array.isArray(val) && Array.isArray(value)) {
+            return matchArray(val, value);
+        } else {
+            return matchObject(val, value);
+        }
+    } else {
+        return isMatch(val, value);
+    }
+}
+
+function containsMatch(array, value) {
+    var len = array.length;
+    var i = -1;
+  
+    while (++i < len) {
+      if (deepMatches(array[i], value)) {
+        return true;
+      }
+    }
+    return false;
+  }
```

Comment:
Is this `}` misformatted? 🤔

Reply:
yeah, I copied from previous file, where were 2 spaces instead of 4